### PR TITLE
Add email campaign statistics

### PR DESCRIPTION
**Pull request by raphaelblum**
Screenshot 2024-03-19 at 11 01 57

Screenshot 2024-03-19 at 11 14 05

Statistics
Screenshot 2024-03-19 at 11 28 26

### CODE REVIEW

File: packages/admin/src/emailCampaigns/statistics/EmailCampaignStatistics.tsx [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+    );
+
+    return (
+        <>
+            <Toolbar>
+                <ToolbarItem>
+                    <IconButton onClick={stackApi?.goBack} size="large">
+                        <ArrowLeft />
+                    </IconButton>
+                </ToolbarItem>
```
Comment (johnnyomair) [inline packages/admin/src/emailCampaigns/statistics/EmailCampaignStatistics.tsx:36]: Please use ToolbarBackButton for this

Comment (raphaelblum) [reply packages/admin/src/emailCampaigns/statistics/EmailCampaignStatistics.tsx]: done

**Review by johnnyomair — DISMISSED** [review]

File: packages/admin/src/emailCampaigns/EmailCampaignsGrid.tsx [hunk 2]
```diff
@@ -202,6 +202,11 @@ export function EmailCampaignsGrid({
                             }
                             refetchQueries={[emailCampaignsQuery]}
                         />
+                        {row.sendingState === "SENT" && (
+                            <IconButton component={StackLink} pageName="statistics" payload={row.id}>
+                                <Statistics color="primary" />
+                            </IconButton>
+                        )}
```
Comment (jamesricky) [inline packages/admin/src/emailCampaigns/EmailCampaignsGrid.tsx:209]: Shouldn't this be shown before the context menu? 
Usually the three dots of the context menu are shown at the end.

**Review by RainbowBunchie — APPROVED** [review]

File: packages/admin/src/emailCampaigns/statistics/PercentageCard.tsx [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+                {renderSkeleton ? (
+                    <Skeleton variant="text" width={80} height={64} />
+                ) : (
+                    <>
+                        {percentage === null ? (
+                            "–"
+                        ) : (
+                            <>
+                                <FormattedNumber value={percentage} style="decimal" minimumSignificantDigits={3} maximumSignificantDigits={3} />%
+                            </>
```
Comment (jamesricky) [inline packages/admin/src/emailCampaigns/statistics/PercentageCard.tsx:29]:
```suggestion
                            <FormattedNumber value={percentage} style="decimal" minimumSignificantDigits={3} maximumSignificantDigits={3} />%
```

File: packages/admin/src/emailCampaigns/statistics/PercentageCard.tsx [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+                    </>
+                )}
+            </Typography>
+            <Typography variant="body2">
+                {renderSkeleton ? (
+                    <Skeleton variant="text" width={100} height={20} />
+                ) : (
+                    <>
+                        <FormattedNumber value={currentNumber} /> / <FormattedNumber value={targetNumber} />
+                    </>
```
Comment (jamesricky) [inline packages/admin/src/emailCampaigns/statistics/PercentageCard.tsx:40]:
```suggestion
                    <FormattedNumber value={currentNumber} /> / <FormattedNumber value={targetNumber} />
```

Comment (raphaelblum) [reply packages/admin/src/emailCampaigns/statistics/PercentageCard.tsx:40]: doesn't work,
there are 2 formattedNumber so the <></> is needed

File: packages/admin/src/emailCampaigns/statistics/PercentageCard.tsx (hunk 3 above)
Comment (raphaelblum) [reply packages/admin/src/emailCampaigns/statistics/PercentageCard.tsx:29]: doesn't work,
there is a FormattedNumber + "%"

File: packages/admin/src/emailCampaigns/EmailCampaignsGrid.tsx (hunk 2 above)
Comment (raphaelblum) [reply packages/admin/src/emailCampaigns/EmailCampaignsGrid.tsx]: okay 👍 
changed in: 581a8d199439708fcf2363e7f4039a6310de2c34

looks like that now: 
Screenshot 2024-03-19 at 12 52 20

**Review by jamesricky — APPROVED** [review]

**Review by RainbowBunchie — APPROVED** [review]

**Review by johnnyomair — APPROVED** [review]

**Pull request merged**